TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -141,7 +142,7 @@ Case cases[] = {
 };
 
 utest::v1::status_t greentea_test_setup(const size_t number_of_cases) {
-    GREENTEA_SETUP((total_ticks + 5) * 3, "timing_drift_auto");
+    GREENTEA_SETUP((total_ticks + 5) * 3, "default_auto");
```

Comment:
Without this change the test fails due to the time_drift_auto host test is not found.
Please, find below the issue:
...
[1475845288.68][HTST][INF] setting timeout to: 45 sec
[1475845288.69][CONN][RXD] {{__host_test_name;timing_drift_auto}}
[1475845288.69][CONN][INF] found KV pair in stream: {{__host_test_name;timing_drift_auto}}, queued...
[1475845288.71][HTST][ERR] host test not detected: timing_drift_auto
…